### DISCUSSION TITLE
Default opacity level is not working

### DISCUSSION THREAD

**Discussion by lionwu** [WindowTop/WindowTop-App#329] [General]
Hello,
The default opacity level is not working,  I set it to 15%, but when I open a new windows and click OPACITY is still 75%. Please check, thank yout!

![123](https://github.com/WindowTop/WindowTop-App/assets/13290191/513a388e-a508-48d4-914c-9d50dd7290b8)

**Comment by gileli121** [comment]
@lionwu 
Hi, sorry for the late reply.
I answered you at https://github.com/WindowTop/WindowTop-App/issues/328 

In addition to my answer, please check for specific configurations on the window.
Click on the arrow button:
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/1c48ac46-21f2-4924-b09d-a608f5c48c36)

If you see only the save button, no configuration is saved for the window.
In this case, it should work as you expect for a new window or after restarting WindowTop.
But if you also see the X button, then I suggest clicking on the X button to delete the configuration
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/74310f7f-e1f7-47b7-b848-2bd61ad37b42)

This will allow the global default opacity (that you configured) to apply when you enable the opacity.
Because you have saved the configuration for the window, it is possible that you experience this behavior.

**Comment by gileli121** [comment]
@lionwu 
From you screenshot I see text in Chinese / Japanese / Korean.
I can't tell.. it looks similar.

Chinese and Korean is available in the app under 
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/e922c2b4-f5c0-46b3-a7e0-264e8a67ef1d)


As for Japanese, this is very new and currently available only in the last beta.
Feel free to test it (in case you Japanese). you can download from here:
https://github.com/WindowTop/WindowTop-App/releases/tag/v5.22.5

Check the beta4 release.
This is the latest beta release with Japanese support.

**Comment by gileli121** [comment]
Answered and fixed
https://github.com/WindowTop/WindowTop-App/issues/328
Closed
